### PR TITLE
Develop offerts and details

### LINKED ISSUE #12: Develop Homepage and detail Models and view models

### PR DESCRIPTION
**Pull request by storres21**
merge between main and the database connection with the homepage and details

### CODE REVIEW

File: Andlet/Andlet/Models/TestFirestoreConnectionView.swift [hunk 1]
```diff
@@ -0,0 +1,83 @@
+import SwiftUI
+import FirebaseFirestore
+
+struct TestFirestoreConnectionView: View {
```
Comment (Daniel7578) [inline Andlet/Andlet/Models/TestFirestoreConnectionView.swift]: Should we keep this view in the Models folder ?

File: Andlet/Andlet/ViewModels/Offers/OfferDetailViewModel.swift [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+        let isAndes = data["is_andes"] as? Bool ?? false
+        let name = data["name"] as? String ?? "Unknown"
+        let phone = data["phone"] as? String ?? "0000000000"
+        let typeUserString = data["type_user"] as? String ?? "landlord"
+        let typeUser = UserType(rawValue: typeUserString) ?? .landlord
+        let favoriteOffers = data["favorite_offers"] as? [Int] ?? []
+
+        return UserModel(
+            id: userEmail,  // Usamos el email como el ID
+            email: email,
```
Comment (Daniel7578) [inline Andlet/Andlet/ViewModels/Offers/OfferDetailViewModel.swift]: If we use the email as the id should we keep the email attribute ?

File: Andlet/Andlet/Models/TestFirestoreConnectionView.swift (hunk 1 above)
Comment (storres21) [reply Andlet/Andlet/Models/TestFirestoreConnectionView.swift]: Ohh no, It was just a test

File: Andlet/Andlet/ViewModels/Offers/OfferDetailViewModel.swift (hunk 2 above)
Comment (storres21) [reply Andlet/Andlet/ViewModels/Offers/OfferDetailViewModel.swift]: You are right, the id should be the email

**Review by Daniel7578 — APPROVED** [review]
Great Job

**Pull request merged**